### PR TITLE
Use AWS_ENDPOINT_URL if specified

### PR DESCRIPTION
**Pull request by gordon-klotho**
This PR adds support for localstack as discovered by their CTO in https://github.com/whummer/localstack-klotho-quickstart/tree/main/patches

### Standard checks

- **Unit tests**: Any special considerations?
- **Docs**: Do we need to update any docs, internal or public?
- **Backwards compatibility**: Will this break existing apps? If so, what would be the extra work required to keep them working?

### CODE REVIEW

File: pkg/lang/javascript/aws_runtime/_keyvalue.ts [hunk 1]
```diff
@@ -51,13 +53,6 @@ const KVStore = new Entity(
     { table: '{{.AppName}}', client: docClient }
 )
 
-import { fs as cloudFS } from './fs'
```
Comment (yuval-klotho) [inline pkg/lang/javascript/aws_runtime/_keyvalue.ts:54]: unrelated cleanup, I assume?

**Review by yuval-klotho — APPROVED** [review]

File: pkg/lang/javascript/aws_runtime/_keyvalue.ts (hunk 1 above)
Comment (gordon-klotho) [reply pkg/lang/javascript/aws_runtime/_keyvalue.ts:54]: Yeah, I figured while I was in the neighborhood I'd leave it cleaner than when I got here :)
